server/file.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const remark = require('remark');
+const lint = require('remark-lint');
+const html = require('remark-html');
+
+function getMarkdown(fileName) {
+  return new Promise((resolve, reject) => {
+    fs.readFile(fileName, (err, markdown) => {
+      if (!err) {
+        resolve(markdown.toString('utf-8'));
+      } else {
+        reject(err.message);
+      }
+    });
+  });
+}
+
+function getFileContent() {
+  return new Promise((resolve, reject) => {
```

Comment:
then please construct this Promise inside `.then((markdown) => {
   return new Promise…
}` to have it be as local and specific as possible